packages/ui/src/tailwind.config.ts

```diff
@@ -111,6 +112,11 @@ const config = {
       },
     },
   },
+  plugins: [
+    plugin(function ({ addVariant }) {
+      addVariant('supports-ios', '@supports (-webkit-touch-callout: none)');
```

Comment:
Nice shout on this! Is your thinking to bring over the CSS from #3897 with these additional `@supports` queries?

I'll likely be off by the time this is merged, but also just wanted to make a point of suggesting to collaborate with Alvish on this 🫡

Reply:
@joe-bell yeah, I followed a bit of your approach there. Is this something you can carry across the line. I think we keep it simple with the support sniff similar to how its working atm.